rules/otp.bzl

```diff
@@ -243,6 +243,86 @@ otp_image = rule(
     },
 )
 
+def _otp_image_header(ctx):
+    output = ctx.actions.declare_file(ctx.attr.name + ".h")
+    args = ctx.actions.args()
+    if not ctx.attr.verbose:
+        args.add("--quiet")
+    args.add("--lc-state-def", ctx.file.lc_state_def)
+    args.add("--mmap-def", ctx.file.mmap_def)
+    if ctx.attr.img_seed:
```

Comment:
since these have defaults I dont think you need the "if"s, and by these I mean `img_seed`, `lc_seed`, `otp_seed`

Reply:
Done.